### ISSUE TITLE
gulp init errors on fresh clone

### ISSUE THREAD

**Issue by jeffbcross** [mlaval/poc-angular-react-native#1]
It appears that the `!create` task is trying to initialize a directory that does not yet exist.

```
$ gulp init
[08:48:55] Using gulpfile ~/Projects/poc-angular-react-native/gulpfile.js
[08:48:55] Starting 'clean'...
[08:48:56] Finished 'clean' after 27 ms
[08:48:56] Starting '!create'...
{ [Error: spawn /bin/sh ENOENT]
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawn /bin/sh',
  path: '/bin/sh',
  spawnargs: [ '-c', 'react-native init ngReactNative' ],
  cmd: '/bin/sh -c react-native init ngReactNative' }
[08:48:56] Finished '!create' after 13 ms
[08:48:56] Starting 'init'...
[08:48:57] Finished 'init' after 1.41 s
```

If I run `gulp init` again, it initializes correctly.

**Comment by mlaval** [comment]
It works fine on a Windows machine :/

**Comment by mlaval** [comment]
Fixed by f1286a781f2e7df40759af91850bd740579fb5c9
